### PR TITLE
fix(docs): Fix broken links in ansible docs

### PR DESCRIPTION
**Pull request by oradwell**
Ansible docs have moved. Updating them so people don't end up on 404 pages

closes #12571

### CODE REVIEW

**Comment by oradwell** [comment]
@thiskevinwang can you take a look at this, please?

**Comment by thiskevinwang** [comment]
Hey @chrisroberts, we currently have the `backport/website` label for backporting to changes to `stable-website`,

But we'd probably want to backport to **N-2**
- `v2.2.19`
- `v2.2.18`
- `v2.2.17`

... which would require creating new `release/2.2.19` `release/2.2.18` `release/2.2.17` branches

File: website/content/docs/provisioning/ansible_intro.mdx [hunk 1]
```diff
@@ -22,7 +22,7 @@ To learn more about Ansible, please consult the [Ansible Documentation Site](htt
 ## The Playbook File
 
 The first component of a successful Ansible provisioner setup is the Ansible playbook which contains the steps that should be run on the guest. Ansible's
-[playbook documentation](https://docs.ansible.com/playbooks.html) goes into great detail on how to author playbooks, and there are a number of [best practices](https://docs.ansible.com/playbooks_best_practices.html) that can be applied to use Ansible's powerful features effectively.
+[playbook documentation](https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html) goes into great detail on how to author playbooks, and there are a number of [tips and tricks](https://docs.ansible.com/ansible/latest/user_guide/playbooks_best_practices.html) that can be applied to use Ansible's powerful features effectively.
```
Comment (oradwell) [inline website/content/docs/provisioning/ansible_intro.mdx:25]: we can avoid redirection on the tips and tricks URL
```suggestion
[playbook documentation](https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html) goes into great detail on how to author playbooks, and there are a number of [tips and tricks](https://docs.ansible.com/ansible/latest/tips_tricks/ansible_tips_tricks.html) that can be applied to use Ansible's powerful features effectively.
```

File: website/content/docs/provisioning/ansible_intro.mdx [hunk 2]
```diff
@@ -22,7 +22,7 @@ To learn more about Ansible, please consult the [Ansible Documentation Site](htt
 ## The Playbook File
 
 The first component of a successful Ansible provisioner setup is the Ansible playbook which contains the steps that should be run on the guest. Ansible's
-[playbook documentation](https://docs.ansible.com/playbooks.html) goes into great detail on how to author playbooks, and there are a number of [best practices](https://docs.ansible.com/playbooks_best_practices.html) that can be applied to use Ansible's powerful features effectively.
+[playbook documentation](https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html) goes into great detail on how to author playbooks, and there are a number of [tips and tricks](https://docs.ansible.com/ansible/latest/tips_tricks/ansible_tips_tricks.html) that can be applied to use Ansible's powerful features effectively.
```
Comment (oradwell) [inline website/content/docs/provisioning/ansible_intro.mdx:25]: we avoid redirection on playbook guide
```suggestion
[playbook documentation](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_intro.html) goes into great detail on how to author playbooks, and there are a number of [tips and tricks](https://docs.ansible.com/ansible/latest/tips_tricks/ansible_tips_tricks.html) that can be applied to use Ansible's powerful features effectively.
```

**Comment by oradwell** [comment]
@chrisroberts @thiskevinwang this is still broken

**Review by thiskevinwang — APPROVED** [review]
👍

**Pull request merged**